dist/index.js

```diff
@@ -5560,7 +5560,8 @@ class GitCommandManager {
                 '--no-tags',
                 '--prune',
                 '--progress',
-                '--no-recurse-submodules'
+                '--no-recurse-submodules',
+                '--verbose'
```

Comment:
how much info will this print? will it help with the exit 1 case?
Should we export `GITHUB_TRACE=1` and `GIT_CURL_VERBOSE=1` when we retry for the last time?